yellowbrick/regressor/base.py

```diff
@@ -24,32 +24,34 @@
 
 
 ## Packages for export
-__all__ = [
-    "RegressionScoreVisualizer",
-]
+__all__ = ["RegressionScoreVisualizer"]
 
 
 ##########################################################################
 ## Regression Visualization Base Object
 ##########################################################################
 
+
 class RegressionScoreVisualizer(ScoreVisualizer):
     """
     Base class for all ScoreVisualizers that evaluate a regression estimator.
 
     The primary functionality of this class is to perform a check to ensure
     the passed in estimator is a regressor, otherwise it raises a
     ``YellowbrickTypeError``.
+
+    .. todo:: enhance the docstrings here and for score
     """
 
-    def __init__(self, model, ax=None, **kwargs):
-        if not isregressor(model):
+    def __init__(self, model, ax=None, fig=None, force_model=False, **kwargs):
+        if not force_model and not isregressor(model):
```

Comment:
Nice! It will be fantastic to have this be consistent across the regressors and classifiers (I know that's already in the works in an upcoming PR 😉 ). Reminder to self for us to add a bit of test coverage of `force_model` when we do the next PR to add tests for the bases of the classifier and regressor modules!